test/src/org/labkey/test/components/targetedms/QCPlotsWebPart.java

```diff
@@ -237,7 +237,8 @@ public String getCurrentEndDate()
         LHRATIO("Light/Heavy Ratio", false),
         TPAREARATIO("Transition/Precursor Area Ratio", true),
         TPAREAS("Transition & Precursor Areas", true),
-        MASSACCURACTY("Mass Accuracy", true);
+        MASSACCURACTY("Mass Accuracy", true),
```

Comment:
eventually, we should change MASSACCURACTY to be MASSACCURACY (lol one of these has a typo in it)